packages/appsync-emulator-serverless/elasticsearchSource.js

```diff
@@ -0,0 +1,33 @@
+const httpSource = require('./httpSource');
+
+const elasticsearchSource = async (
+  endpoint,
+  { path = '', operation = 'POST', params = {} },
+) => {
+  const esResult = await httpSource(endpoint, {
+    // ES accepts GET/HEAD operations with a body,
+    // but fetch forbids it.
+    method:
+      ['HEAD', 'GET'].includes(operation) !== -1 &&
```

Comment:
I think we want:

`['HEAD', 'GET'].includes(operation) &&`

?

Reply:
ups, yes